src/ciflow-bot.ts

```diff
@@ -275,8 +291,11 @@ export class CIFlowBot {
       ?.map(label => label.name);
     this.owner = this.ctx.payload?.repository?.owner?.login;
     this.repo = this.ctx.payload?.repository?.name;
-    if (!CIFlowBot.allowed_repos.includes(`${this.owner}/${this.repo}`)) {
-      return false;
```

Comment:
Not exactly. For repositories that do not have `.github/pytorch_probot.yml` or does not have issue define, `loadIssue` would return an empty object, which means it will not be deployed to a single user there.